src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt

```diff
@@ -347,6 +349,23 @@ abstract class FixupSession(
           }
         }
 
+        // It is safe to use selectionRange here, because it is set in the constructor
```

Comment:
What if the edit itself adds or removes lines?
Is that reflected by that selection?

Reply:
I've just tested this situation manually, and this property doesn't seem to change. Nevertheless, the range being sent to the agent in `textDocument/didChange` is correct (i.e. increased).